run

```diff
@@ -1,6 +1,7 @@
-#!/usr/bin/env python
+#!/usr/bin/env sh
 
-import websockify
+BASE_DIR="$(dirname "$(realpath "$0")")"
 
-if __name__ == '__main__':
-	websockify.websocketproxy.websockify_init()
+cd "$BASE_DIR"
+
+python -m websockify $@
```

Comment:
This is missing double quotes around `"$@"`, which breaks invocation with arguments including spaces. Fixed in #405.

I recommend always using [ShellCheck](https://www.shellcheck.net/) to scan for common shell scripting errors like this.

Reply:
Thanks for the feedback, I'll keep that in mind.